src/display/api.js

```diff
@@ -251,7 +256,20 @@ function getDocument(src) {
   params.rangeChunkSize = params.rangeChunkSize || DEFAULT_RANGE_CHUNK_SIZE;
   params.CMapReaderFactory = params.CMapReaderFactory || DOMCMapReaderFactory;
   params.ignoreErrors = params.stopAtErrors !== true;
-  params.pdfBug = params.pdfBug === true;
+
+  const pdfBugEnabled = params.pdfBug === true;
+  params.pdfBug = pdfBugEnabled;
+  // To simplify debugging, always force-enable `fontExtraProperties` when
```

Comment:
Given the limited usefulness outlined in the commit messages, does the data actually provide helpful insights when debugging font problems? I'm wondering why we would want this in debug mode.

Reply:
*Please refer to the comment above.*

Reply:
Although I suppose that we could remove the special-casing here, if you think that this code is unnecessarily confusing, and simply set `fontExtraProperties` in https://github.com/mozilla/pdf.js/blob/0400109b8763e53cf657d72fbf1bdab2df9927b9/web/app.js#L310-L314 instead?